presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDriver.java

```diff
@@ -138,4 +137,22 @@ public Logger getParentLogger()
         // TODO: support java.util.Logging
         throw new SQLFeatureNotSupportedException();
     }
+
+    private OkHttpClient newHttpClient()
+    {
+        OkHttpClient.Builder builder = new OkHttpClient.Builder()
+                .addInterceptor(userAgent(DRIVER_NAME + "/" + DRIVER_VERSION));
+
+        String javaVersion = StandardSystemProperty.JAVA_VERSION.value();
```

Comment:
There's `JavaVersion.current()`. As far as I know, "java.version" should never be null, so you should be able to use that one instead.